frontend/src/metabase/query_builder/components/expressions/analytics.ts

```diff
@@ -0,0 +1,8 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+
+export const trackColumnCombineViaShortcut = () => {
+  trackSchemaEvent("question", "1-0-4", {
+    event: "column_combine_via_shortcut",
+    custom_expressions_used: ["concat"],
+  });
```

Comment:
**suggestion/question (not-blocking):**

Does it make sense to populate other properties of the event as well here?
It seems like the following could make sense (if we have the info available):
- `visualization_type`
- `database_id`
- ... I guess we don't have information about the `question_id` or something like that, do we?

Reply:
I think `database_id` is the only one that is trivial to find and pass. Is that fine?

Reply:
Yeah sure - let's start with that one 🙂 .

Just to check: I had the assumption that the `column_combine_via_shortcut` would only happen in table vizualizations but that seems not to hold, does it?